Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN cargo build
 
 
 # Build Node app
-FROM node:16-bullseye
+FROM node:18-bookworm
```

Comment:
Should be 18-bullseye, unless we land https://github.com/chairemobilite/transition/pull/781 first
(This is why the docker build fails)